src/replaceCustomValues.ts

```diff
@@ -221,22 +317,15 @@ export function getElementSubmissionValue({
         continue
       }
 
-      const result = getElementSubmissionValue({
+      const result = getElementSubmissionValueById({
         elementId,
-        propertyName,
         formElements: element.elements,
         submission:
           newSubmissionData as SubmissionTypes.S3SubmissionData['submission'],
-        formatDate,
-        formatDateTime,
-        formatTime,
-        formatNumber,
-        formatCurrency,
       })
       if (result) {
-        return result.value !== undefined || result.value !== null
-          ? result
-          : undefined
+        unknown = result.unknownValue
+        formElement = result.formElement
       }
```

Comment:
can we return here to prevent further interations?

Reply:
And then we dont need a return after the iterations?

Reply:
nup

Reply:
oh wait nevermind